app/templates/components/email-input.hbs

```diff
@@ -1,7 +1,15 @@
-<label for="email">User Name</label>
+{{!-- <label for="email">User Name</label>
```

Comment:
Unused code? Rip it out! Thats what version control is for you can always go back and look.